homeassistant/components/fronius/sensor.py

```diff
@@ -591,13 +591,15 @@ async def async_setup_entry(
     SensorEntityDescription(
         key="capacity_maximum",
         name="Capacity maximum",
-        native_unit_of_measurement=ENERGY_WATT_HOUR,
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
```

Comment:
This sensor represents a capacity, not usage. I don't think that matches the use of the device class.